recordium/config.py

```diff
@@ -43,29 +35,40 @@ class _Config(object):
         'USER_ALLOWED': None,
         'COL_ORDER': []
     }
+    _need_save = 0
```

Comment:
This attribute should not be a class one, but a instance one (set it inside __init__). Also, name it "_needs_save" or "_save_needed"